p4cmd/p4cmd.py

```diff
@@ -407,21 +408,24 @@ def get_shelved_files(self):
 
         return files_and_cl
 
-    def add_or_edit_files(self, file_list=[], changelist="default"):
+    def add_or_edit_files(self, file_list=None, changelist="default"):
```

Comment:
I think we should actually just remove the default value for `file_list`, since you shouldn't be able to call this function without providing a file or file list anyway